lib/graphs/abstract_digraph.nit

```diff
@@ -0,0 +1,583 @@
+# Implementations for representing directed graphs, also called digraphs.
+#
+# Currently, there are two classes
+module abstract_digraph
+
+# Interface for digraphs
+abstract class AbstractDigraph[V]
```

Comment:
Yes. `HashMapDigraph` or maybe just `HashDigraph` as it rely on a working and efficient implementation of `hash` in vertices. Map is important but maybe not that important to be in the name (it should be in the doc)